tests/unit/avro/test_serde.py

```diff
@@ -78,11 +78,19 @@ def test_serialize_and_deserialize():
     assert len(test_records) == 1
     assert test_records[0] == user
 
+    # Ensure serialization / deserialization via generators
+    # works as expected.
     test_buffer = BytesIO(avro_blob)
     test_meta, test_generator = deserialize(test_buffer)
     assert isinstance(test_meta['avro.schema'], bytes)
     assert isinstance(test_generator, types.GeneratorType)
 
+    # Reform the generator by re-encoding the original, passed
+    # as part of a batch.
+    test_generator_blob = serialize(USER_SCHEMA, [test_generator])
+    _, test_generator = deserialize(test_generator_blob)
```

Comment:
Should we check that `test_generator` is what we expect here? Or we just check that we succeeded in deserializing it without errors?

Reply:
The block at see the block two lines below this one.

Reply:
Oh, indeed, I just was embarrassed by rewriting the `test_generator` and haven't checked its usage.
Sorry.
LGTM